parlai/crowdsourcing/tasks/acute_eval/util.py

```diff
@@ -191,7 +191,7 @@ def _check_file_contents(
             results_folder=results_folder, file_suffix=file_suffix
         )
         with open(file_path) as f:
-            contents = f.read()
+            contents = f.read().rstrip('\n') + '\n'
```

Comment:
Okay, this works - maybe there can be a comment saying why this is needed (i.e. to be insensitive to whitespace linting issues?)